cabal.project

```diff
@@ -37,13 +37,13 @@ optional-packages: vendored/*/*.cabal
 tests: true
 
 package *
-  -- ghc 8.10 cannot build ghc-lib 9.2 with --haddock
-  -- ghc-options: -haddock
+  -- ghc 8.10.7 fails to build ghc-lib-9.2.2
```

Comment:
probably shouldn't have diffs in this file

Reply:
spot on! reverted